web/controllers/github_version_controller.ex

```diff
@@ -0,0 +1,21 @@
+defmodule Healthlocker.GithubVersionController do
+  use Healthlocker.Web, :controller
+
+  def index(conn, _params) do
+    # leaving IO.inspect / puts in for debugging till further notice
+    IO.inspect(System.cmd("pwd", []))
```

Comment:
@katbow yeah, leaving the `IO.inspect` are in for now. will remove when deployment is working _seamlessly_. 👍